modules/StaticModule/StaticModuleTrait/ModuleStylesTrait.php

```diff
@@ -79,6 +79,7 @@ public static function module_styles( $args ) {
 							'selector'  => $args['orderClass'],
 							'attr'      => $attrs['css'] ?? [],
 							'cssFields' => StaticModule::custom_css(),
+							'orderClass' => $args['orderClass'],
```

Comment:
***Note a blocker***
You said in description "enhancing the modular styling approach." can you please explain what does this enhance.  As I am not aware of the feature?

Reply:
It's just decorative text. let me remove it as you are confused

Reply:
What I mean is that we are reaching something for sure; we just need to mention what it is exactly.

Reply:
@mohmed98 I already removed this text as it's created confusion.